examples/pico/logger/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.helidon.examples.pico</groupId>
+        <artifactId>helidon-examples-pico-project</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.helidon.examples.pico.logger</groupId>
+    <artifactId>helidon-examples-pico-logger-project</artifactId>
+    <name>Helidon Pico Examples - Logger</name>
+
+    <packaging>pom</packaging>
+
+    <properties>
+        <version.lib.guice>5.1.0</version.lib.guice>
+
+        <!-- pico supports legacy javax.inject and javax.annotations in tooling -->
+        <javax.injection.version>1</javax.injection.version>
+        <javax.annotations.version>1.3.2</javax.annotations.version>
+
+<!--        <checkstyle.skip>true</checkstyle.skip>-->
```

Comment:
Leftover?

Reply:
will fix in next push.

Reply:
I am assuming you are using javax dependencies for a reason (as stated in the README) so yes, this is ok. We want to limit the exposure to just this example.